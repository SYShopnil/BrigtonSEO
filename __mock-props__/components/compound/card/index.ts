import { eventHistoryCardData } from './event-history';
import { testimonialData, testimonialWithoutImageData } from './testimonial';
import {
  speakerCardNormalData,
  speakerCardTransparentData,
  teamData,
} from './person';
import { courseCardData } from './course';
import { upcomingCardData } from './upcoming';
import { pricingCardLargeData, pricingCardSmallData } from './pricing-card';
import { buyTicketCardData } from './buy-ticket';

export {
  testimonialData,
  pricingCardLargeData,
  pricingCardSmallData,
  courseCardData,
  upcomingCardData,
  speakerCardNormalData,
  speakerCardTransparentData,
  teamData,
  eventHistoryCardData,
  testimonialWithoutImageData,
  buyTicketCardData,
};
