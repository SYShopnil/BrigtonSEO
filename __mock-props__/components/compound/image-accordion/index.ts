import {
  IImageAccordion,
  IImageAccordionCard,
  SectionThemeEnum,
} from '@src/types/compound';

export const imageAccordionCardData: IImageAccordionCard = {
  image:
    'https://res.cloudinary.com/dktw59lc2/image/upload/v1670331012/BrightonSEO/image-accordion/home/brightonseo_d4nh2j.png',
  title: 'BrightonSEO',
  link: 'https://www.google.com/',
  description:
    'Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis.',
};

export const imageAccordionCardsData: IImageAccordionCard[] = [
  { ...imageAccordionCardData },
  {
    ...imageAccordionCardData,
    title: 'Trainers',
    image:
      'https://res.cloudinary.com/dktw59lc2/image/upload/v1670331013/BrightonSEO/image-accordion/home/trainers_udeetj.png',
  },
  {
    ...imageAccordionCardData,
    title: 'Training Sessions',
    image:
      'https://res.cloudinary.com/dktw59lc2/image/upload/v1670331013/BrightonSEO/image-accordion/home/training-sessions_deaidv.png',
  },
  {
    ...imageAccordionCardData,
    title: 'London Training',
    image:
      'https://res.cloudinary.com/dktw59lc2/image/upload/v1670331012/BrightonSEO/image-accordion/home/london-training_ebqrhk.png',
  },
  {
    ...imageAccordionCardData,
    title: 'In-Company Training',
    image:
      'https://res.cloudinary.com/dktw59lc2/image/upload/v1670331012/BrightonSEO/image-accordion/home/in-company-training_ysgg2q.png',
  },
];

export const imageAccordionData: IImageAccordion = {
  sectionTheme: SectionThemeEnum.Blue,
  sectionHeading: {
    subTitle: 'About the events',
    title: 'Welcome to the Greatest SEO Conference 2022',
  },
  cards: imageAccordionCardsData,
};
