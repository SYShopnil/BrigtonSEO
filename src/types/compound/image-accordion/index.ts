import { SectionThemeEnum } from '../common';

interface ISectionHeading {
  subTitle: string;
  title: string;
}

export interface IImageAccordionCard {
  image: string;
  title: string;
  link: string;
  description: string;
}

export interface IImageAccordion {
  sectionHeading: ISectionHeading;
  cards: IImageAccordionCard[];
  sectionTheme: SectionThemeEnum;
}
