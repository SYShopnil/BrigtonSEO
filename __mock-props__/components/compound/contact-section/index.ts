/**
 * Internal Imports
 */
import { IContactSection } from '@src/types/compound';
import { formData } from '@root/__mock-props__/components/root/form';

// contactSectionWithoutAddressData
export const contactSectionWithoutAddressData: IContactSection = {
  sectionHeading: {
    title: 'Host For Fringe Events',
    description:
      'Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit.',
  },
  contact: {
    phone: {
      label: 'Phone',
      value: '+456 9876 5432',
    },
  },
  formData: formData,
};

// contactSectionWithAddressData
export const contactSectionWithAddressData: IContactSection = {
  sectionHeading: {
    subTitle: 'Contact Us',
    title: 'We Here to help',
  },
  contact: {
    address: {
      label: 'Office',
      value:
        'Plus X Innovation Hub, c/o SiteVisibility Lewes Road, Brighton, BN2 4GL',
    },
    phone: {
      label: 'Phone',
      value: '+456 9876 5432',
    },
  },
  formData: formData,
};
