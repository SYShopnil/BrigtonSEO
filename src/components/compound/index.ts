import { ContactSection } from './contact-section';
import { ListContentSection } from './list-content-section';
import BuyTicketCard from './card/buy-ticket';
import { HeroGeneral, HeroHome } from './hero-section';
import CourseCard from './card/course';
import PricingCard from './card/pricing';
import { Testimonial } from './card/testimonial';
import Person from './card/person';
import UpcomingCard from './card/upcoming';
import FooterSection from './footer-section';
import { TestimonialSection } from './testimonial-section';
import { EventHistoryCard } from './card/event-history';
import { TextAccordion } from './textAccordion';
import { ImageAccordion } from './image-accordion';

export {
  ContactSection,
  CourseCard,
  FooterSection,
  ListContentSection,
  Testimonial,
  HeroGeneral,
  BuyTicketCard,
  PricingCard,
  Person,
  EventHistoryCard,
  UpcomingCard,
  TestimonialSection,
  HeroHome,
  TextAccordion,
  ImageAccordion,
};
