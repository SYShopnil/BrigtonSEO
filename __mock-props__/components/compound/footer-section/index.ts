import { IFooterSection } from '@src/types/compound';
import { IconEnum } from '@src/types/root';

export const footerSectionData: IFooterSection = {
  headingInfo: {
    title: 'Want something extra?',
    subTitle: 'Subscribe to newsletter',
    buttonText: 'Subscribe',
    buttonLink: '/subscribe',
    linkOpenNewTab: false,
  },
  contactInfo: {
    address:
      'c/o SiteVisibility, Plus X Innovation Hub, Lewes Road Brighton, BN2 4GL - 01273 805525',
    phoneNumberOne: '+965 6554 6598',
    phoneNumberTwo: '+965 6554 6599',
    email: 'brightonseo@gmail.com',
  },
  columnOne: {
    title: 'Company',
    linkFields: [
      {
        label: 'About Us',
        url: '/about-us',
      },
      {
        label: 'Tickets',
        url: '/tickets',
      },
      {
        label: 'Speakers',
        url: '/speakers',
      },
      {
        label: 'Training',
        url: '/training',
      },
      {
        label: 'Sponsors',
        url: '/sponsors',
      },
      {
        label: 'Career',
        url: '/career',
      },
      {
        label: 'Contact',
        url: '/contact',
      },
    ],
  },
  columnTwo: {
    title: 'Support',
    linkFields: [
      {
        label: 'FAQs',
        url: '/faqs',
      },
      {
        label: 'Code of Conduct',
        url: '/code-of-conduct',
      },
      {
        label: 'Privacy Policy',
        url: '/privacy-policy',
      },
      {
        label: 'Terms & Conditions',
        url: '/terms-and-conditions',
      },
      {
        label: 'Unsubscribe',
        url: '/unsubscribe',
      },
    ],
  },
  socialColumn: {
    title: 'Follow Us',
    socialLinks: [
      {
        platform: IconEnum.Facebook,
        url: 'https://www.facebook.com',
      },
      {
        platform: IconEnum.Twitter,
        url: 'https://www.twitter.com',
      },
      {
        platform: IconEnum.Instagram,
        url: 'https://www.instagram.com',
      },
      {
        platform: IconEnum.Youtube,
        url: 'https://www.youtube.com',
      },
    ],
  },
};
