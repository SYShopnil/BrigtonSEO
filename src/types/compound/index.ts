import { IContactSection, IContact } from './contact-section';
import {
  IListItem,
  IListContentSection,
  IContentHead,
  ListStyleEnum,
  IListContentCtaButton,
} from './list-content-section';
import { ITestimonialSection } from './testimonial-section';
import { IAuthor } from './common';
import {
  IHeroGeneral,
  HeroBgColorEnum,
  IHeroHome,
  EventInfoThemeEnum,
  IHeroLogo,
  IInfoCount,
} from './hero-section';
import { ITestimonial } from './card/testimonial';
import {
  IPricingCard,
  PriceBoxColorSchemeEnum,
  PriceBoxSizeEnum,
} from './card/pricing';
import { IPerson, ITeam, IPersonTransparent } from './card/person';
import { IFooterSection } from './footer-section';
import { IBuyTicketCard } from './card/buy-ticket';
import { IEventHistoryCard } from './card/event-history';
import { ICourseCard, ITag, TagBackgroundEnum } from './card/course';
import type { IUpcomingCard } from './card/upcoming';
import { UpcomingCardThemeEnum } from './card/upcoming';
import { IEventDate, IEventInfo, UserIconSizeEnum } from './common';
import { SectionThemeEnum } from './common';
import { ITextAccordion, ITextAccordionList, IFAQ } from './textAccordion';
import { IImageAccordion, IImageAccordionCard } from './image-accordion';

export type {
  IContactSection,
  IContact,
  IListItem,
  IListContentSection,
  IContentHead,
  ITestimonial,
  IBuyTicketCard,
  ICourseCard,
  ITag,
  IHeroGeneral,
  IPerson,
  IUpcomingCard,
  IFooterSection,
  IListContentCtaButton,
  IHeroHome,
  IEventInfo,
  IHeroLogo,
  IInfoCount,
  IPricingCard,
  ITextAccordion,
  ITextAccordionList,
  IFAQ,
  ITeam,
  IPersonTransparent,
  IEventHistoryCard,
  ITestimonialSection,
  IAuthor,
  IEventDate,
  IImageAccordion,
  IImageAccordionCard,
};

export {
  HeroBgColorEnum,
  UpcomingCardThemeEnum,
  ListStyleEnum,
  EventInfoThemeEnum,
  PriceBoxColorSchemeEnum,
  PriceBoxSizeEnum,
  TagBackgroundEnum,
  UserIconSizeEnum,
  SectionThemeEnum,
};
