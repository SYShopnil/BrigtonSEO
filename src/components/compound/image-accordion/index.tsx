import { SectionHeading, Swiper, SwiperItem } from '@src/components/root';
import { useIsDomReady, useViewport } from '@src/hooks';
import { IImageAccordion, SectionThemeEnum } from '@src/types/compound';
import { SectionHeadingColorschemesEnum } from '@src/types/root';
import Styles from './index.module.scss';
import { Card } from './card';
import { useState } from 'react';

/**
 *
 * @param {object}  IImageAccordion
 * @returns JSX.Element
 */
export const ImageAccordion = ({
  sectionHeading,
  cards,
  sectionTheme,
}: IImageAccordion) => {
  const [activeIndex, setActiveIndex] = useState(0);
  const { width } = useViewport();
  const isDomReady = useIsDomReady();

  //  section heading color scheme based on section theme
  const sectionHeadingColorScheme =
    sectionTheme === SectionThemeEnum.Blue
      ? SectionHeadingColorschemesEnum.Blue
      : SectionHeadingColorschemesEnum.Skyblue;

  /**
   * Tailwind classess
   */
  const accordionWrapper = `rounded-[.625rem] overflow-hidden`;

  return (
    <section
      className={`section-padding-primary bg-gray`}
      data-testid="image-accordion-section"
    >
      <div className="container">
        <div className="max-w-[986px]">
          <SectionHeading
            {...sectionHeading}
            colorScheme={sectionHeadingColorScheme}
            hasBottomSpacing={true}
            isCenter={false}
          />
        </div>

        {/* Carousel for mobile to tablet device */}
        {isDomReady && width <= 991 && (
          <div className={`${accordionWrapper}`}>
            <Swiper
              showItems={{ sm: 1.2, md: 2.2, lg: 2, xl: 3, '2xl': 4 }}
              spaceBetween={{ sm: 0, md: 0, lg: 0, xl: 0, '2xl': 0 }}
            >
              {cards.map((card, index) => (
                <SwiperItem key={index}>
                  {({ isActive }) => (
                    <Card {...card} isLarge={false} isActive={isActive} />
                  )}
                </SwiperItem>
              ))}
            </Swiper>
          </div>
        )}
      </div>

      {/* Image accordion for Large to Above Device  */}
      {isDomReady && width >= 992 && (
        <div className={`${Styles['custom-container']} container`}>
          <div
            className={`${accordionWrapper} ${Styles['image-accordion-grid']} image-accordion-grid`}
          >
            {cards.map((card, index) => (
              <Card
                onHover={() => {
                  setActiveIndex(index);
                }}
                key={index}
                {...card}
                isLarge={true}
                isActive={index === activeIndex}
              />
            ))}
          </div>
        </div>
      )}
    </section>
  );
};
