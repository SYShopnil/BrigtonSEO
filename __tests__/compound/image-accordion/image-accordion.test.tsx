import { imageAccordionData } from '@root/__mock-props__/components/compound/image-accordion';
import { ImageAccordion } from '@src/components/compound/image-accordion';
import { render } from '@testing-library/react';

/**
 * Test if the image accordion section correctly
 */
it(`renders image accordion section correctly`, () => {
  const { getByTestId } = render(<ImageAccordion {...imageAccordionData} />);
  const section = getByTestId('image-accordion-section');
  expect(section).toBeInTheDocument();
  expect(section).toMatchSnapshot();
});
