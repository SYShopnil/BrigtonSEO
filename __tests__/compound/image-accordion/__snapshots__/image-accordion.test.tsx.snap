// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders image accordion section correctly 1`] = `
<section
  class="section-padding-primary bg-gray"
  data-testid="image-accordion-section"
>
  <div
    class="container"
  >
    <div
      class="max-w-[986px]"
    >
      <div
        class="section-heading blue  mb-[1.875rem] md:mb-[3.25rem]"
        data-testid="section-heading"
      >
        <span
          class="sub-title block text-[1.125rem] font-medium leading-[1.5] md:text-[1.5rem]  md:leading-[1.2] capitalize"
        >
          About the events
        </span>
        <h2
          class="section-title text-[1.75rem] font-medium leading-[1.3] md:text-[3rem] md:font-bold md:leading-[1.1] capitalize"
        >
          Welcome to the Greatest SEO Conference 2022
        </h2>
      </div>
    </div>
  </div>
  <div
    class="custom-container container"
  >
    <div
      class="rounded-[.625rem] overflow-hidden image-accordion-grid image-accordion-grid"
    >
      <div
        class=" active large  card  relative overflow-hidden"
      >
        <span
          class="card-overlay absolute inset-0 z-[1]"
        />
        <div
          class="inline-flex absolute w-full h-full"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              alt="BrightonSEO"
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
            />
            <noscript />
          </span>
        </div>
        <h3
          class="title-inactive  text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
        >
          <span>
            BrightonSEO
          </span>
        </h3>
        <div
          class="content absolute left-0 bottom-0 p-4 lg:p-[3.75rem] z-10"
        >
          <div
            class="content-inner"
          >
            <div
              class="content-inner-inner"
            >
              <h3
                class=" title text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
              >
                <a
                  href="https://www.google.com/"
                >
                  BrightonSEO
                </a>
              </h3>
              <p
                class="description text-sm lg:text-[1.125rem] lg:leading-[1.6] mt-4 text-white"
              >
                Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="  large  card  relative overflow-hidden"
      >
        <span
          class="card-overlay absolute inset-0 z-[1]"
        />
        <div
          class="inline-flex absolute w-full h-full"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              alt="Trainers"
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
            />
            <noscript />
          </span>
        </div>
        <h3
          class="title-inactive  text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
        >
          <span>
            Trainers
          </span>
        </h3>
        <div
          class="content absolute left-0 bottom-0 p-4 lg:p-[3.75rem] z-10"
        >
          <div
            class="content-inner"
          >
            <div
              class="content-inner-inner"
            >
              <h3
                class=" title text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
              >
                <a
                  href="https://www.google.com/"
                >
                  Trainers
                </a>
              </h3>
              <p
                class="description text-sm lg:text-[1.125rem] lg:leading-[1.6] mt-4 text-white"
              >
                Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="  large  card  relative overflow-hidden"
      >
        <span
          class="card-overlay absolute inset-0 z-[1]"
        />
        <div
          class="inline-flex absolute w-full h-full"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              alt="Training Sessions"
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
            />
            <noscript />
          </span>
        </div>
        <h3
          class="title-inactive  text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
        >
          <span>
            Training Sessions
          </span>
        </h3>
        <div
          class="content absolute left-0 bottom-0 p-4 lg:p-[3.75rem] z-10"
        >
          <div
            class="content-inner"
          >
            <div
              class="content-inner-inner"
            >
              <h3
                class=" title text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
              >
                <a
                  href="https://www.google.com/"
                >
                  Training Sessions
                </a>
              </h3>
              <p
                class="description text-sm lg:text-[1.125rem] lg:leading-[1.6] mt-4 text-white"
              >
                Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="  large  card  relative overflow-hidden"
      >
        <span
          class="card-overlay absolute inset-0 z-[1]"
        />
        <div
          class="inline-flex absolute w-full h-full"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              alt="London Training"
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
            />
            <noscript />
          </span>
        </div>
        <h3
          class="title-inactive  text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
        >
          <span>
            London Training
          </span>
        </h3>
        <div
          class="content absolute left-0 bottom-0 p-4 lg:p-[3.75rem] z-10"
        >
          <div
            class="content-inner"
          >
            <div
              class="content-inner-inner"
            >
              <h3
                class=" title text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
              >
                <a
                  href="https://www.google.com/"
                >
                  London Training
                </a>
              </h3>
              <p
                class="description text-sm lg:text-[1.125rem] lg:leading-[1.6] mt-4 text-white"
              >
                Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="  large  card  relative overflow-hidden"
      >
        <span
          class="card-overlay absolute inset-0 z-[1]"
        />
        <div
          class="inline-flex absolute w-full h-full"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              alt="In-Company Training"
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
            />
            <noscript />
          </span>
        </div>
        <h3
          class="title-inactive  text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
        >
          <span>
            In-Company Training
          </span>
        </h3>
        <div
          class="content absolute left-0 bottom-0 p-4 lg:p-[3.75rem] z-10"
        >
          <div
            class="content-inner"
          >
            <div
              class="content-inner-inner"
            >
              <h3
                class=" title text-[1.5rem] font-semibold leading-[1.4] md:text-[1.75rem] text-white lg:text-[3rem] lg:font-bold lg:leading-[1.1]"
              >
                <a
                  href="https://www.google.com/"
                >
                  In-Company Training
                </a>
              </h3>
              <p
                class="description text-sm lg:text-[1.125rem] lg:leading-[1.6] mt-4 text-white"
              >
                Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
